### ISSUE TITLE
Nextcloud doesnt start because of an unexpected character 0x00 in Group/Manager.php

### ISSUE THREAD

**Issue by wiryfuture** [nextcloud-snap/nextcloud-snap#2227]
#### Describe the bug
Nextcloud fails to do anything; cannot use occ or interact with the website,
When i try to do `nextcloud.occ status`, I get a massive stack trace off of 
```
An unhandled exception has been thrown:
ParseError: syntax error, unexpected character 0x00 in /snap/nextcloud/32113/htdocs/lib/private/Group/Manager.php:88
Stack trace:
#0 /snap/nextcloud/32113/htdocs/lib/composer/composer/ClassLoader.php(428): Composer\Autoload\includeFile()
#1 /snap/nextcloud/32113/htdocs/lib/private/Server.php(476): Composer\Autoload\ClassLoader->loadClass()
#2 /snap/nextcloud/32113/htdocs/lib/private/AppFramework/Utility/SimpleContainer.php(162): OC\Server->OC\{closure
}()
#3 /snap/nextcloud/32113/htdocs/3rdparty/pimple/pimple/src/Pimple/Container.php(122): OC\AppFramework\Utility\Sim
pleContainer->OC\AppFramework\Utility\{closure}()
#4 /snap/nextcloud/32113/htdocs/lib/private/AppFramework/Utility/SimpleContainer.php(129): Pimple\Container->offs
etGet()
#5 /snap/nextcloud/32113/htdocs/lib/private/ServerContainer.php(136): OC\AppFramework\Utility\SimpleContainer->qu
ery()
#6 /snap/nextcloud/32113/htdocs/lib/private/AppFramework/Utility/SimpleContainer.php(57): OC\ServerContainer->que
ry()
#7 /snap/nextcloud/32113/htdocs/lib/private/Server.php(923): OC\AppFramework\Utility\SimpleContainer->get()
#8 /snap/nextcloud/32113/htdocs/lib/private/AppFramework/Utility/SimpleContainer.php(162): OC\Server->OC\{closure}()
#9 /snap/nextcloud/32113/htdocs/3rdparty/pimple/pimple/src/Pimple/Container.php(122): OC\AppFramework\Utility\SimpleContainer->OC\AppFramework\Utility\{closure}()
#10 /snap/nextcloud/32113/htdocs/lib/private/AppFramework/Utility/SimpleContainer.php(129): Pimple\Container->offsetGet()
#11 /snap/nextcloud/32113/htdocs/lib/private/ServerContainer.php(136): OC\AppFramework\Utility\SimpleContainer->query()
#12 /snap/nextcloud/32113/htdocs/lib/private/AppFramework/Utility/SimpleContainer.php(57): OC\ServerContainer->query()
#13 /snap/nextcloud/32113/htdocs/lib/private/AppFramework/Utility/SimpleContainer.php(184): OC\AppFramework\Utility\SimpleContainer->get()
#14 /snap/nextcloud/32113/htdocs/lib/private/AppFramework/Utility/SimpleContainer.php(162): OC\AppFramework\Utility\SimpleContainer->OC\AppFramework\Utility\{closure}()
#15 /snap/nextcloud/32113/htdocs/3rdparty/pimple/pimple/src/Pimple/Container.php(118): OC\AppFramework\Utility\Si
mpleContainer->OC\AppFramework\Utility\{closure}()
#16 /snap/nextcloud/32113/htdocs/lib/private/AppFramework/Utility/SimpleContainer.php(129): Pimple\Container->off
setGet()
#17 /snap/nextcloud/32113/htdocs/lib/private/ServerContainer.php(136): OC\AppFramework\Utility\SimpleContainer->q
uery()
#18 /snap/nextcloud/32113/htdocs/lib/private/AppFramework/Utility/SimpleContainer.php(57): OC\ServerContainer->qu
ery()
#19 /snap/nextcloud/32113/htdocs/lib/private/Server.php(1991): OC\AppFramework\Utility\SimpleContainer->get()
#20 /snap/nextcloud/32113/htdocs/lib/private/legacy/OC_App.php(386): OC\Server->getAppManager()
#21 /snap/nextcloud/32113/htdocs/lib/private/AppFramework/Bootstrap/Coordinator.php(91): OC_App::getEnabledApps()
#22 /snap/nextcloud/32113/htdocs/lib/base.php(661): OC\AppFramework\Bootstrap\Coordinator->runInitialRegistration
()
#23 /snap/nextcloud/32113/htdocs/lib/base.php(1111): OC::init()
#24 /snap/nextcloud/32113/htdocs/console.php(48): require_once('...')
#25 /snap/nextcloud/32113/htdocs/occ(11): require_once('...')
#26 {main}
```
#### To Reproduce
1. Have a working nextcloud instance
2. I think the last thing i did was revert to a previous version and then refresh back to the more recent one or something after i was getting a different bug?

#### Expected behavior
Nextcloud works like it usually does.

#### Screenshots
Website is just a blank http500 error like this: 
![image](https://user-images.githubusercontent.com/50517631/200426785-87770ec8-2140-40f8-a726-bf35caf6f055.png)

#### OS/snapd/snap version
nextcloud  24.0.6snap2+git2.acd980b (it was on 24.0.6snap2 when i had this issue so tried a more recent version and issue remains)
snap    2.56.2-4.fc36
snapd   2.56.2-4.fc36
series  16
fedora  36
kernel  6.0.5-200.fc36.x86_64

#### Logs
[nextcloud-log.md](https://github.com/nextcloud-snap/nextcloud-snap/files/9955999/nextcloud-log.md) (maximum is 65536 characters)

**Comment by kyrofa** [comment]
Dupe of #2209, your kernel is incompatible with snaps.

**Comment by kyrofa** [comment]
It sounds like the fix has landed in v6.0.7, FYI.
